src/components/Card/CardSecondary/CardSecondary.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen } from "@testing-library/react";
+import { ThemeProvider } from "@/theme";
+import { CardSecondary, CardProps } from "./CardSecondary";
+
+describe("CardSecondary Component", () => {
+  const renderCard = (props: CardProps) =>
+    render(
+      <ThemeProvider theme="light">
+        <CardSecondary {...props} />
+      </ThemeProvider>
+    );
+
+		it("should render the title", () => {
+      const title = "Test card component";
+      renderCard({
+				title,
+				image: "warning",
+				description: "",
+				infoUrl: "",
+				infoText: ""
+			});
+
+      expect(screen.getByText(title)).toBeDefined();
```

Comment:
secondary assertions like `toBeDefined` `toBeInDocument` etc can have weird side effects and make the code more complex.

a way to simplify and standardize the code could be to try to stick with one simple assertion (`toEqual`)

this could be:
`expect(screen.getAllByText(title).length).toEqual(1);`